spec/index.html

```diff
@@ -839,10 +839,11 @@ <h2>Triple-term based reification</h2>
   <p>
     A triple term can be the object of a triple whose predicate is 
     <a href="#ch_reifies"><code>rdf:reifies</code></a>. In such cases, the resulting triple is referred 
-    to as a <em>reifying triple</em>, and its subject is termed a <em>reifier</em>. 
+    to as a <a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying triple</a>, and its subject is 
+    termed a <a data-cite="RDF12-CONCEPTS#dfn-reifier">reifier</a>. 
```

Comment:
I know the URL display highlights these terms. Various things can cause the URL display *not* to differ from surrounding text, so the `<em>` remains worthwhile.
```suggestion
    to as a <em><a data-cite="RDF12-CONCEPTS#dfn-reifying-triple">reifying triple</a></em>, and its subject is 
    termed a <em><a data-cite="RDF12-CONCEPTS#dfn-reifier">reifier</a></em>. 
```